### ISSUE TITLE
Batch normalization: Accelerating deep network training by reducing internal covariate shift

### ISSUE THREAD

**Issue by leemik3** [yskim0/One-Day-One-Paper#44]
## Batch normalization: Accelerating deep network training by reducing internal covariate shift
link : http://proceedings.mlr.press/v37/ioffe15.html
Ioffe, Sergey, and Christian Szegedy. "Batch normalization: Accelerating deep network training by reducing internal covariate shift." International conference on machine learning. PMLR, 2015.

---

### Introduction
기존) **Internal Covariate Shift**(internal node의 분포 변화) 문제로 인한 vanishing gradient problem이 있었음. 
제안) 이를 해결하면 training 속도가 빨라지기 때문에 해결하기 위한 메커니즘으로 **Batch Normalization** 제안

### Batch Normalization의 효과
- 파라미터 크기, 초깃값 에 따른 기울기의 의존성을 줄임으로써 gradient flow에 긍정적 영향을 주어 높은 학습률을 쓸 수 있게 함
- 모델을 정규화해서 drop out 의 필요성을 낮춤
- 네트워크가 saturated 상태에 갇히지 않게 함으로써 'saturaring nonlinearity' 를 쓸 수 있게 함

### Batch Normalization 
stochastic gradient descent에서 각 미니배치 별로 평균과 분산을 계산하여 정규화하고, scale and shift 과정을 거쳐서 batch normalizing transform을 수행한다. 
![image](https://user-images.githubusercontent.com/57309696/128850892-544d786e-eb00-4af0-addb-3920440ff118.png)
